xl/player/gst/missing_plugin.py

```diff
@@ -0,0 +1,171 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+#
+# The developers of the Exaile media player hereby grant permission
+# for non-GPL compatible GStreamer and Exaile plugins to be used and
+# distributed together with GStreamer and Exaile. This permission is
+# above and beyond the permissions granted by the GPL license by which
+# Exaile is covered. If you modify this code, you may extend this
+# exception to your version of the code, but you are not obligated to
+# do so. If you do not wish to do so, delete this exception statement
+# from your version.
+
+'''
+This module serves as frontend for GStreamer Plugin Base Utils (GstPbutils) for
+installing missing plugins.
+
+See also:
+* https://github.com/quodlibet/quodlibet/blob/master/quodlibet/quodlibet/player/gstbe/player.py#L548
+* https://git.gnome.org/browse/totem/tree/src/backend/bacon-video-widget-gst-missing-plugins.c
+
+This module must be a singleton, i.e. no instances are allowed. Thus, there is
+no class.
+
+Created on 25.04.2017
+
+@author: Christian Stadelmann
+'''
+
+import logging
+
+from xl.nls import gettext as _
+
+import gi
+from gi.repository import Gst
+gi.require_version('GstPbutils', '1.0')
```

Comment:
Ok, moving to `main.py`. This way we will see the error earlier, because most people hopefully never run the code I added.